packages/components/app/styles/components/dropdown.scss

```diff
@@ -14,6 +14,20 @@
 
 $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
 
+// DROPDOWN
+
+// TODO: which one of these?
+// .hds-dropdown--display-inline { // same as flight-icon
+// .hds-dropdown--is-inline-block {
```

Comment:
I would probably go with `--is-inline-block` if only to match the component argument, but no strong feelings.

Reply:
👍